libs/chlorophyll/scss/common/_focus.scss

```diff
@@ -42,25 +58,24 @@
     $color: transparentize($color, 0.35);
   }
   &:focus {
-    outline: $color solid 2px;
-    outline-offset: 2px;
+    outline: $color solid .125rem;
+    outline-offset: .125rem;
   }
 
   &:focus:not(:focus-visible) {
     outline: 0;
   }
 
   &:focus-visible {
-    outline: $color solid 2px;
-    outline-offset: 2px;
+    outline: $color solid .125rem;
+    outline-offset: .125rem;
```

Comment:
I don't think this will work properly for Safari, we've seen that behavior when trying `focus-visible` stuff there.

Reply:
Patrik, how long ago was this?

> This is great news! Safari has long been the missing link here, but with the up-coming Safari 15.4, :focus-visible will have full support across modern browsers.
https://www.tempertemper.net/blog/safari-focus-visible-and-accessibility

Safari have version 16.3 right now. 15.4 was released March 15, 2022.

Reply:
It was earlier this month, I think we tested both Safari 15 and 16. We ended up using something like this:

```scss
@mixin focus-outline($color: $black, $size: 0.125rem) {
  &:focus {
    outline: $size solid $color;
    outline-offset: $size;
  }
  &:focus:not(:focus-visible) {
    box-shadow: none;
    outline: 0;
  }
}

```